openpype/hosts/max/api/lib.py

```diff
@@ -497,3 +565,167 @@ def get_plugins() -> list:
+    elif preview_preset == "Customize":
+        preview_preset = "userdefined"
+    else:
+        preview_preset = preview_preset.lower()
+    preview_preset_option = f"vpPreset:#{preview_preset}"
+    job_args.append(preview_preset_option)
+    viewport_texture = instance.data.get("vpTexture", True)
+    if viewport_texture:
+        viewport_texture_option = f"vpTexture:{viewport_texture}"
+        job_args.append(viewport_texture_option)
+
+    auto_play_option = "autoPlay:false"
+    job_args.append(auto_play_option)
+
+    job_str = " ".join(job_args)
+    log.debug(job_str)
+
+    return job_str
+
+
+def publish_preview_sequences(staging_dir, filename,
+                              startFrame, endFrame,
+                              percentSize, ext):
+    """publish preview animation by creating bitmaps
+    ***For 3dsMax Version <2024
+
+    Args:
+        staging_dir (str): staging directory
+        filename (str): filename
+        startFrame (int): start frame
+        endFrame (int): end frame
+        percentSize (int): percentage of the resolution
+        ext (str): image extension
+    """
+    # get the screenshot
+    resolution_percentage = float(percentSize) / 100
+    res_width = rt.renderWidth * resolution_percentage
+    res_height = rt.renderHeight * resolution_percentage
+
+    viewportRatio = float(res_width / res_height)
+
+    for i in range(startFrame, endFrame + 1):
+        rt.sliderTime = i
+        fname = "{}.{:04}.{}".format(filename, i, ext)
+        filepath = os.path.join(staging_dir, fname)
+        filepath = filepath.replace("\\", "/")
+        preview_res = rt.bitmap(
+            res_width, res_height, filename=filepath)
+        dib = rt.gw.getViewportDib()
+        dib_width = float(dib.width)
+        dib_height = float(dib.height)
+        renderRatio = float(dib_width / dib_height)
+        if viewportRatio <= renderRatio:
+            heightCrop = (dib_width / renderRatio)
+            topEdge = int((dib_height - heightCrop) / 2.0)
+            tempImage_bmp = rt.bitmap(dib_width, heightCrop)
+            src_box_value = rt.Box2(0, topEdge, dib_width, heightCrop)
+        else:
+            widthCrop = dib_height * renderRatio
+            leftEdge = int((dib_width - widthCrop) / 2.0)
+            tempImage_bmp = rt.bitmap(widthCrop, dib_height)
+            src_box_value = rt.Box2(0, leftEdge, dib_width, dib_height)
+        rt.pasteBitmap(dib, tempImage_bmp, src_box_value, rt.Point2(0, 0))
+
+        # copy the bitmap and close it
+        rt.copy(tempImage_bmp, preview_res)
+        rt.close(tempImage_bmp)
+
+        rt.save(preview_res)
+        rt.close(preview_res)
+
+        rt.close(dib)
+
+        if rt.keyboard.escPressed:
+            rt.exit()
+    # clean up the cache
+    rt.gc(delayed=True)
+
+def publish_preview_animation(instance, staging_dir, filepath,
+                              startFrame, endFrame, review_camera):
+    """Publish Reivew Animation
```

Comment:
```suggestion
    """Render camera review animation
```

Would also be good if the docstring would mention what it outputs - is it a video file? is it a sequence? if it's a sequence what type of filepath do I need to provide? will it append frame numbers? what defines the output format/extension? etc.

Library functions preferably are clear functions with the simplest entry point to usage - as opposed to requiring an `instance` which could have ANY `data` defined.